AIMSICD/src/main/java/com/secupwn/aimsicd/data/DefaultDataTransaction.java

```diff
@@ -0,0 +1,277 @@
+package com.secupwn.aimsicd.data;
+
+import com.secupwn.aimsicd.data.model.DefaultLocation;
+import com.secupwn.aimsicd.data.model.GpsLocation;
+import com.secupwn.aimsicd.data.model.SmsDetectionString;
+
+import io.realm.Realm;
+
+public class DefaultDataTransaction implements Realm.Transaction {
```

Comment:
Depending on how this data gets updated it will be more efficient to create a pre-populated Realm file and ship that with the app, see e.g: https://github.com/realm/realm-java/blob/master/examples/migrationExample/src/main/java/io/realm/examples/realmmigrationexample/MigrationExampleActivity.java#L56-Lundefined

It does require a bit more setup though as you need way for your build system to create and package the Realm file with the app.